builder/comp-builder.nix

```diff
@@ -162,8 +167,12 @@ let
       if configureAllComponents
         then ["--enable-tests" "--enable-benchmarks"]
         else ["${haskellLib.componentTarget componentId}"]
-    ) ++ [ "$(cat ${configFiles}/configure-flags)"
-    ] ++ commonConfigureFlags);
+    ) ++ [ "$(sed 's/--dependency=\\(.*\\)+\\(.*\\)/--dependency=\\1/' ${configFiles}/configure-flags)"
```

Comment:
Needs a comment, that explains what (and why) we are running sed on the configure-flags. E.g. why are they wrong by construction, and what do we need them to look like, and why?